generators/app/templates/src/redux/store.js

```diff
@@ -21,7 +23,7 @@ middlewares.push(thunk);
 
 // TODO Add this if you need it
 /* ------------- Analytics Middleware ------------- */
-// middlewares.push(AnalyticsMiddleware);
+// Middlewares.push(AnalyticsMiddleware);
```

Comment:
mmm 🤔 Does eslint yield with `// _middlewares.push`?

Reply:
same issue as above... should I remove that rule that sets every comment as capitalized?

Reply:
IMHO will be a bit more comfortable 😊 What do you think ?